LASR-at-Home/Base
# Phase 2 - First iteration

This PR provides an initial implementation of Phase 2. The main functionality here is the loop of requesting orders and then checking their validity. Checking items are actually "on the counter" is implemented as in #21 for now.

Testing instructions:
- Choose a table to use as the counter.
- If not already taken, collect the cuboid points for the table, fill these in `config/test_make_order.yaml`.
- Position TIAGo in front of the table, and adjust him (e.g. head, torso) such that he has a clear view of the table. This is the case currently, because I'm not sure what assumptions we can make about the counter.
- Find some objects that are in the `coco` dataset.
- Adjust the "order" in `config/test_make_order.yaml`.
- Place some objects on the counter (experiment with different setups, e.g. the correct order, missing items, superfluous items)
- `roslaunch coffee_shop make_order.launch`

Expected behavior:
- Missing/superfluous items are noted by TIAGo, and the order is asked to be fixed.
- If the order is correct, then nothing should happen.
